test/base/testing_brave_browser_process.cc

```diff
@@ -0,0 +1,63 @@
+/* Copyright (c) 2021 The Brave Software Team. Distributed under the MPL2
+ * license. This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <utility>
+
+#include "brave/components/brave_shields/browser/ad_block_service.h"
+#include "brave/components/ipfs/buildflags/buildflags.h"
```

Comment:
`gn_check` complains about just this include, which seems wrong:

```
ERROR at //brave/test/base/testing_brave_browser_process.cc:9:11: Can't include this header from here.
#include "brave/components/ipfs/buildflags/buildflags.h"
          ^--------------------------------------------
The target:
  //brave/test:brave_unit_tests
is including a file from the target:
  //brave/components/ipfs/buildflags:buildflags

It's usually best to depend directly on the destination target.
In some cases, the destination target is considered a subcomponent
of an intermediate target. In this case, the intermediate target
should depend publicly on the destination to forward the ability
to include headers.

Dependency chain (there may also be others):
  //brave/test:brave_unit_tests -->
  //brave/browser/browsing_data:browsing_data --[private]-->
  //brave/components/ipfs/buildflags:buildflags
```

Reply:
Hmmm... just checked `//brave/test/BUILD.gn` and it seems like indeed you need to add `//brave/components/ipfs/buildflags` to `deps` for the `//brave/tests/brave_unit_tests` target.

Reply:
awesome! that fixed it :smile: